include/quicly/frame.h

```diff
@@ -207,7 +208,7 @@ typedef struct st_quicly_ack_frame_t {
     uint64_t ack_delay;
     uint64_t num_gaps;
     uint64_t ack_block_lengths[257];
```

Comment:
Please use QUICLY_MAX_ACK_FRAME_RANGE (or the macro renamed from that) for calculating ack_block_lengths.